pallets/common/src/traits/mod.rs

```diff
@@ -49,6 +49,7 @@ pub trait CommonConfig: frame_system::Config + permissions::Config {
         + Debug
         + Unsigned
         + From<u128>
+        + Into<u128>
```

Comment:
My idea was to just drop this associated type entirely in favour of using `polymesh_primitives::Balance`. I think this can have positive knock-on effects throughout the code-base, allowing us to drop more type parameters and so on. But we can leave this as a separate task after merging this PR.